symbols/overlay29.yml

```diff
@@ -8536,6 +8536,75 @@ overlay29:
         Called when the in-dungeon "others" menu is open. Does not return until the menu is closed.
         
         return: Always 0
+    - name: CheckVariousStatuses
+      address:
+        EU: 0x23016DC
+        NA: 0x2300AD0
+        JP: 0x2301D50
```

Comment:
These NA/JP addresses aren't correct. I checked them in Ghidra, and they end up in the middle of random functions.

I can see that you computed the NA/JP symbols here by adding the EU offset (0x23016DC) to the (NA - EU == 0x234FA24 - 0x2350630) and (JP - EU == 0x2350CA4 - 0x2350630) shifts from `OthersMenu`. In general, naively trying to compute the offsets of symbols in a different version like this is not reliable. The different versions can sometimes change/insert/delete code in between your symbol and the "reference" symbol you're using, which means the interversion shifts will not be the same between the two symbols.

We have more reliable ways of automatically determining the proper offsets for different versions, so please don't fill these in with "guesses" if you aren't sure they're correct. Having less information is better than having wrong information, and we can autofill the other versions afterwards. `pmdsky-debug` allows you to omit the addresses for versions you aren't sure about, so you can just add the address in the version you're working on (EU). For example:

```yml
    - name: CheckVariousStatuses
      address:
       EU: 0x23016DC
```

is a perfectly valid entry on its own.

Reply:
Sorry about that, I've removed them.